src/server/location.rs

```diff
@@ -124,12 +124,25 @@ impl LocationResult {
         headers.get(GOOG_LOC_HEADER).is_some()
     }
 
-    pub fn region(&self) -> String {
-        self.subdivision.clone().unwrap_or_default()
+    pub fn region(&self) -> Option<String> {
+        if self.country() == "US" {
+            return Some(self.subdivision.clone().unwrap_or_default());
+        }
+        None
     }
 
     pub fn country(&self) -> String {
-        self.country.clone().unwrap_or_default()
+        self.country.clone().unwrap_or_default().to_uppercase()
+    }
+
+    pub fn dma(&self) -> String {
```

Comment:
nit: consider keeping this as u16 to match [actix-web-location](https://github.com/mozilla-services/contile/pull/219) which we'll switch to shortly

Reply:
I might make this an `Option<u16>` so that the "None" case is still captured.